# DP: install apport exception hander in site.py

--- a/Lib/site.py	2004-07-20 12:28:28.000000000 +1000
+++ b/Lib/site.py	2006-11-09 09:28:32.000000000 +1100
@@ -393,6 +393,14 @@
     # this module is run as a script, because this code is executed twice.
     if hasattr(sys, "setdefaultencoding"):
         del sys.setdefaultencoding
+    # install the apport exception handler if available
+    try:
+        import apport_python_hook
+    except ImportError:
+        pass
+    else:
+        apport_python_hook.install()
+
 
 main()
 
